luminoso_api/upload.py

```diff
@@ -118,9 +124,18 @@ def main():
             lang, reader_name = item.split('=', 1)
             reader_dict[lang] = reader_name
 
+    date_format = args.date_format.lower()
```

Comment:
What if they pass a strftime format string?  Then `lower()`ing it is super-sad-panda, no?